packages/nimble-components/src/table-column/date-text/group-header-view/index.ts

```diff
@@ -0,0 +1,47 @@
+import { DesignSystem } from '@microsoft/fast-foundation';
+import type { TableNumberFieldValue } from '../../../table/types';
+import { TableColumnTextGroupHeaderViewBase } from '../../text-base/group-header-view';
+import { template } from '../../text-base/group-header-view/template';
+import { styles } from '../../text-base/group-header-view/styles';
+import type { TableColumnDateTextColumnConfig } from '..';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-table-column-date-text-group-header': TableColumnDateTextGroupHeaderView;
+    }
+}
+/**
+ * The group header view for displaying date/time fields as text.
+ */
+export class TableColumnDateTextGroupHeaderView extends TableColumnTextGroupHeaderViewBase<
+TableNumberFieldValue,
+TableColumnDateTextColumnConfig
+> {
+    private groupHeaderValueChanged(): void {
+        if (typeof this.groupHeaderValue === 'number') {
+            const formatter = new Intl.DateTimeFormat(undefined, {
```

Comment:
I would prefer to kick this can down the road a little bit, to when I add configuration. I'll have a better sense then for how much complication it adds vs benefit.

Reply:
Seems like it would be easy to make it private static (pretty much as good as it gets) and then handle it being dependent on state in the future.